### PR TITLE
Update multiple files

### PR DESCRIPTION
**Pull request by zellman01**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,10 +32,6 @@
     "url": "^0.11.0",
     "websocket": "^1.0.17"
   },
-  "optionalDependencies": {
```
Comment (DeathlyPlays) [inline package.json]: Why?

Comment (DeathlyPlays) [reply package.json]: Btw side note, on main they fixed the modules thing so just update to main again

Comment (zellman01) [reply package.json]: Look at the file: It's still there. It's just for some reason that github says that line was removed. It was removed then moved up to the second line, which should work like normal

**Comment by zellman01** [comment]
That should work fine

**Pull request merged**